### PR TITLE
Initial setup for Agenda.

### PR DESCRIPTION
**Pull request by lukewar**
Please take a look at my part. Merge it if you like it.
@azubala @literator @paweldudek

### CODE REVIEW

File: TDDWorkshop/Classes/Agenda/AgendaViewController.swift [hunk 1]
```diff
@@ -5,16 +5,38 @@
 
 import UIKit
 
-class AgendaViewController: UIViewController {
+class AgendaViewController: UITableViewController {
+
+    var agendaProvider: AgendaProviding!
```
Comment (literator) [inline TDDWorkshop/Classes/Agenda/AgendaViewController.swift:10]: Why no `let`?

**Comment by literator** [comment]
Looks good :)

File: TDDWorkshop/Classes/Agenda/AgendaViewController.swift (hunk 1 above)
Comment (paweldudek) [inline TDDWorkshop/Classes/Agenda/AgendaViewController.swift:10]: I assume it's something you might want to change later on, looks fine to me.

Comment (lukewar) [inline TDDWorkshop/Classes/Agenda/AgendaViewController.swift:10]: exactly it is changed.

**Comment by lukewar** [comment]
Awesome, then please merge.

**Pull request merged**